rllib/algorithms/algorithm.py

```diff
@@ -620,11 +620,18 @@ def setup(self, config: AlgorithmConfig) -> None:
             validate_env=self.validate_env,
             default_policy_class=self.get_default_policy_class(self.config),
             config=self.config,
-            num_env_runners=self.config.num_env_runners,
+            num_env_runners=0 if self.config.input_ else self.config.num_env_runners,
             local_env_runner=True,
             logdir=self.logdir,
         )
 
+        # Ensure remote workers are initially in sync with the local worker.
+        self.workers.sync_weights(inference_only=True)
```

Comment:
dumb question: Why do we need this additional sync? Which (currently existing) sync does this replace?

Reply:
I think this is a left-over from a missed merge with master? Could you check?